app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        __class__.people[self.name] = self
```

Comment:
Use another approach
```suggestion
        Person.people[self.name] = self
```